Sources/ARSAL_Time.c

```diff
@@ -122,7 +163,12 @@ int ARSAL_Time_GetLocalTime (struct timespec *res, struct tm *localTime)
         /* No else --> Do nothing if res is NULL */
         if (localTime != NULL)
         {
+#ifdef _WIN32
+            struct tm *tmpLocalTime = localtime(&(ts.tv_sec));
```

Comment:
Not thread safe. Using functions like FileTimeToLocalFileTime and FileTimeToSystemTime + the 1/1/1970 ref as a FILETIME it should be possible to have something safer
static const FILETIME EPOCH = {something const and that can be precomputed, ref of FILETIME  is 1/1/1601)

FILETIME ft = EPOCH + ts
FileTimeToLocalFileTime(&ft, &ft2);
FileTimeToSystemTime (&ft2, &st)
convert SYSTEMTIME to struct tm

EPOCH precomputaion:
		memset(&st1, 0, sizeof(st1));
		st1.wDay = 1;
		st1.wMonth = 1;
		st1.wYear = 1970;
		SystemTimeToFileTime(&st1, &ft1);